db/migrate/20150515093642_create_purchase_suppliers.rb

```diff
@@ -0,0 +1,10 @@
+class CreatePurchaseSuppliers < ActiveRecord::Migration
+  def change
+    create_table :purchase_suppliers, options: 'ENGINE=InnoDB DEFAULT CHARSET=utf8' do |t|
+      t.string :name # поставщик
+      t.integer :inn, :limit=> 8 # инн
```

Comment:
Use the new Ruby 1.9 hash syntax.<br>Surrounding space missing for operator '=>'.